ckanext/ldap/controllers/user.py

```diff
@@ -321,14 +329,15 @@ def _ldap_search(cnx, filter_str, attributes, non_unique=u'raise'):
 
     '''
     try:
-        res = cnx.search_s(config[u'ckanext.ldap.base_dn'], ldap.SCOPE_SUBTREE,
+        res = cnx.search_s(base_dn_str, ldap.SCOPE_SUBTREE,
                            filterstr=filter_str, attrlist=attributes)
+        '''res = cnx.search_s(None, ldap.SCOPE_SUBTREE, filterstr=filter_str, attrlist=attributes)
+        '''
     except ldap.SERVER_DOWN:
         log.error(u'LDAP server is not reachable')
         return None
     except ldap.OPERATIONS_ERROR as e:
-        log.error(
-            u'LDAP query failed. Maybe you need auth credentials for performing searches? Error returned by the server: ' + e.info)
+        log.error(u'LDAP query failed. Maybe you need auth credentials for performing searches?')
```

Comment:
Can the error info be added back in? Or another way or getting more information about the error for the logs?

Reply:
The e.info does not exist. I was getting a Notice in the error log, that says the info atribute is not available when it did enter in that except. 
I don't know how I could get more information to de logs.

```

Reply:
The ldap library's doc isn't all that clear but I think from my quick look the `info` attribute is optional and dependant on whether the server returned an extra info or not. Perhaps we could check if there is an `info` attribute available or not and then log it or not based on that?